pools/zergpool.go

```diff
@@ -0,0 +1,58 @@
+package pools
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/vertcoin-project/one-click-miner-vnext/util"
+)
+
+var _ Pool = &Zergpool{}
+
+type Zergpool struct {
+	Address           string
+	LastFetchedPayout time.Time
+	LastPayout        uint64
+}
+
+func NewZergpool(addr string) *Zergpool {
+	return &Zergpool{Address: addr}
+}
+
+func (p *Zergpool) GetPendingPayout() uint64 {
+	jsonPayload := map[string]interface{}{}
+	err := util.GetJson(fmt.Sprintf("http://api.zergpool.com:8080/api/wallet?address=%s", p.Address), &jsonPayload)
+	if err != nil {
+		return 0
+	}
+	vtc, ok := jsonPayload["unpaid"].(float64)
+	if !ok {
+		return 0
+	}
+	vtc *= 100000000
+	return uint64(vtc)
+}
+
+func (p *Zergpool) GetStratumUrl() string {
+	return "stratum+tcp://verthash.na.mine.zergpool.com:4534"
```

Comment:
Does NA stand for north america? If they have an EU server, i think you should add two pools to allow users to pick the one closest to them. Otherwise european users have an unfair disadvantage in terms of latency.

Reply:
Updated Zergpool StratumURL to Anycast IP